modules/ROOT/pages/_partials/commons/common-new-logging-api.adoc

```diff
@@ -0,0 +1,634 @@
+// {param-module} grabs the correct platform name from the related _set_page_context_for_[platform] 
+== Upgrading to the New CBL Logging API
+
+CAUTION: Use of the deprecated and new Logging API at the same time is not supported.
+
+You can find information about the new Couchbase Lite Logging API introduced in Couchbase Lite 3.2.2.
+
+ifndef::is-c[]
+
+For information about the now deprecated earlier version of the Logging API, see xref:3.1@couchbase-lite:{param-module}:troubleshooting-logs.adoc[]
+
+endif::is-c[]
+
+== LogSinks
+
+Couchbase Lite 3.2.2 introduces a new Logging API.
+
+The first thing to note is that the three destinations for logs have been renamed as `LogSinks`, in keeping with common source/sink terminology. 
+
+The `FileLogSink`, the `ConsoleLogSink` and the `CustomLogSink` are all installed in an instance of `LogSinks`. 
+
+All three log sinks are initialized by default. 
+If you want to stop all logging to one of them, though, you can set the sink to null.
+
+For example, you can stop logging to the FileLogSink like this:
+
+`LogSinks.get().setFile(null)`
```

Comment:
This is very specific to Java / Android and as only console log sink is enabled by default, I think we could just say something like :

```
To enable a specific type of LogSink, create a LogSink object of that type, set its minimum log level and domains, and assign it to LogSinks. To disable a LogSink, set it to null or use a LogSink with LogLevel.NONE.
```

The code snippet for disabling the log sink should be shown here, we will need the code snippet for each platform. Maybe the current code snippet for setting each type of log sink is enough?

Reply:
Reworded as suggested.